src/tree/OpenUrlAzExtTreeItem.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtParentTreeItem, GenericTreeItem } from "vscode-azureextensionui";
+import { openExternal } from "../utils/openExternal";
+import { getThemedIconPath } from "./IconPath";
+import { OpenUrl } from "./OpenUrlTreeItem";
+
+export class OpenUrlAzExtTreeItem extends GenericTreeItem implements OpenUrl {
```

Comment:
I'm not sure I understand why this was split apart from `OpenUrlTreeItem` and why two classes are needed.

Reply:
I tried to use the existing one but that require to pass a parent tree item and have dependency on AzExtTreeItem. Hence a new one that has no dependency on AzExtTreeItem and also allows using icons from vscode.